randomsub.go

```diff
@@ -119,6 +123,14 @@ func (rs *RandomSubRouter) Publish(msg *Message) {
 	}
 
 	if len(rspeers) > RandomSubD {
+		target := RandomSubD
+		sqrt := int(math.Ceil(math.Sqrt(float64(rs.size))))
```

Comment:
This assumes the size of the network is fixed for the lifetime of the router - what's the suggested use of randomsub in the case where the network size is varying?

Reply:
It is more of a hint actually. Note that we don't recommend using randomsub in real networks, it was done more as an experiment.